ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -875,13 +875,7 @@ def test_torch_multi_dot(
     dtype_and_input=_get_dtype_and_matrix(batch=True),
 )
 def test_torch_pinv(
-    *,
-    dtype_and_input,
-    frontend,
-    test_flags,
-    fn_tree,
-    backend_fw,
-    on_device,
+    *, dtype_and_input, frontend, test_flags, fn_tree, backend_fw, on_device, hermitian
```

Comment:
Please draw the hermitian value from random boolean value generator like below:
```
@handle_frontend_test(
    fn_tree="torch.linalg.pinv",
    dtype_and_input=_get_dtype_and_matrix(batch=True),
    hermitian=st.booleans(),
)
def test_torch_pinv(
    *,
    dtype_and_input,
    frontend,
    test_flags,
    fn_tree,
    backend_fw,
    on_device,
    hermitian,
):
    input_dtype, x = dtype_and_input
    helpers.test_frontend_function(
        input_dtypes=input_dtype,
        backend_to_test=backend_fw,
        frontend=frontend,
        test_flags=test_flags,
        fn_tree=fn_tree,
        on_device=on_device,
        input=x[0],
        atol=1e-02,
        rtol=1e-02,
        hermitian=hermitian
    )
```

and please fix the test failures that appear after that